src/Mai/mappings/CamAave.ts

```diff
@@ -0,0 +1,74 @@
+import { log } from "@graphprotocol/graph-ts";
+
+import { Transfer as TransferEvent } from "../../../generated/MaiCamAavecamAAVE/MaiCamAave";
+
+import { ZERO_ADDRESS, ZERO_BI } from "../../utils/constants";
+import { handleCamToken } from "./handlers";
+import { MaiCamAave } from "../../../generated/MaiCamAavecamAAVE/MaiCamAave";
+
+export function handleTransfer(event: TransferEvent): void {
+  let from = event.params.from;
+  let to = event.params.to;
+  let value = event.params.value;
+
+  // only interested on mints and burns.
+  if (from != ZERO_ADDRESS && to != ZERO_ADDRESS) {
+    return;
+  }
+
+  if (to == ZERO_ADDRESS) {
+    // burn
+    value = value.neg();
```

Comment:
I think `value` is not used. Please check and clean up (in all files). Maybe more can be removed.

Reply:
As I'm getting the data from the directly from the contract, that variable wasn't used anymore. It's removed now.